plugins/inputs/redis_sentinel/redis_sentinel.go

```diff
@@ -0,0 +1,442 @@
+package redis_sentinel
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"net/url"
+	"strconv"
+	"strings"
+	"sync"
+
+	"github.com/go-redis/redis"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type RedisSentinel struct {
+	Servers  []string `toml:"servers"`
+	Password string   `toml:"password"`
```

Comment:
The password setting is still here and just vanished from the documentation. Can you please fix it?

Reply:
@srebhan Completely missed this! The password is now properly parsed from the provided DSN string. See https://github.com/influxdata/telegraf/pull/10042/commits/4c78cad94dcf65095ff18ddea5be6a699d4e7f27.